frontend/src/components/utils/tags/TagList.tsx

```diff
@@ -0,0 +1,53 @@
+import { api, PaginatedResponse, useBetterPaginatedQuery, useBetterQuery } from "../../../services/apiService";
+import React from "react";
+import { Tag as TagComponent } from "./Tag";
+import { Tag } from "../../../models/tag";
+
+export enum Models {
```

Comment:
I think this name is too generic and could lead to naming issues
Maybe `TagModels`?